cabal-install/src/Distribution/Client/CmdOutdated.hs

```diff
@@ -212,8 +214,8 @@ outdatedOptions _showOrParseArgs =
 -------------------------------------------------------------------------------
 
 -- | Entry point for the 'outdated' command.
-outdatedAction :: OutdatedFlags -> [String] -> GlobalFlags -> IO ()
-outdatedAction OutdatedFlags{..} _targetStrings globalFlags = do
+outdatedAction :: (ProjectFlags, OutdatedFlags) -> [String] -> GlobalFlags -> IO ()
+outdatedAction (ProjectFlags{..}, OutdatedFlags{..}) _targetStrings globalFlags = do
```

Comment:
Is a lot the fields from `ProjectFlags` used there now or will be in the future (I guess that's the plan for the future and this is the crux of the PR?)? If it's only one field and no future plans, could you list it instead of the two dots? E.g., this would help me understand this during the review, so possibly it would help others reading the code.

Reply:
`ProjectFlags` has only two fields, I don't feel like this is significant. Afaict, this should not have a single addition to the help message, at most the role of `--project-file` is a bit harder to grasp.

I think you are rather thinking about `NixStyleFlags` which has a ton of fields.

Reply:
Got it. How about `ProjectFlags{theFieldUsedHere}`?

Reply:
Done!

Reply:
Thank you. Now I see without asking.